src/intl/translated-messages.json

```diff
@@ -175,7 +182,7 @@
     "createANewVm": "Eine neue virtuelle Maschine erstellen",
     "createVm": "VM erstellen",
     "currentlyInsertedIsoInCdDrive": "Aktuell eingelegte ISO im CD-Laufwerk",
-    "definedMemory": "Definierter Arbeitsspeicher",
+    "definedMemory": "Arbeitsspeicher",
```

Comment:
You just changed the German translation in Zanata and re-pulled, correct?

Reply:
Yes, this change was done in Zanata first and then pulled to the code.